package.json

```diff
@@ -6,7 +6,6 @@
   "scripts": {
     "test": "npm run lint && npm run build && mocha dist/test/test-*.js",
     "cover": "npm run build && istanbul cover _mocha -- dist/test/test-*.js",
-    "prepublish": "npm run test",
```

Comment:
I use this such that it actually builds and tests, when I type `npm publish`.
I'd like to keep this behavior intact.

Reply:
ok